Description: Compilation from the upstream code depends on being in
  the git repository to get the correct git-revision to be shown when
  running the program. This patch puts in the git revision from the
  correct upstream.
  It must be updated every time upstream changes!
Forwarded: doesn't make sense upstream
Author: Ruben Undheim <ruben.undheim@gmail.com>
Index: yosys/Makefile
===================================================================
--- yosys.orig/Makefile	2014-10-28 21:24:31.983412258 +0100
+++ yosys/Makefile	2014-10-28 21:24:28.447412191 +0100
@@ -48,7 +48,7 @@
 endif
 
 YOSYS_VER := 0.3.0+$(shell test -d .git && { git log --author=clifford@clifford.at --oneline ca125bf41.. | wc -l; })
-GIT_REV := $(shell git rev-parse --short HEAD 2> /dev/null || echo UNKNOWN)
+GIT_REV := f9c096e
 OBJS = kernel/version_$(GIT_REV).o
 
 # set 'ABCREV = default' to use abc/ as it is
